ast/check.go

```diff
@@ -213,7 +213,6 @@ func (tc *typeChecker) checkRule(env *TypeEnv, as *AnnotationSet, rule *Rule) {
 	var tpe types.Type
 
 	if len(rule.Head.Args) > 0 {
```

Comment:
This is a minor diff that got left in after a local rebase (when the check moved from `check.go` -> `parser.go`).